gr-qtgui/lib/sink_c_impl.h

```diff
@@ -1,6 +1,6 @@
 /* -*- c++ -*- */
 /*
- * Copyright 2008,2009,2011,2012,2014 Free Software Foundation, Inc.
+ * Copyright 2008,2009,2011,2012,2014,2018 Free Software Foundation, Inc.
```

Comment:
Not sure if I have to fix all copyright headers of files I touched ...

Reply:
Luckily, especially for minor or automated fixes, "latest" copyright dates are of lesser importance.